assets/sass/_planetPortrait.scss

```
@@ -1,29 +1,38 @@
 .main{
-  .planet-portrait {
-    height: 60vh;
-    width: 100%;
-    background-color: red;
+
+  max-width: 1100px;
+  background-color: paleturquoise;
+  margin: 0 auto;
+  height: 100vh;
+
+  .planet-portrait{ 
+    height: 60%;
+    width: 80%;
```

```suggestion
    width: 100%;
```

Acá no deberías ponerlo menor al 100%, porque esto debería ocupar todo el ancho del .main container, no?